examples/config.yaml

```diff
@@ -0,0 +1,88 @@
+input:
+  - BoundingBoxDataPointsFilter:
+      xMin: -1.5
+      xMax: 0.5
+      yMin: -1
+      yMax: 1
+      zMin: -1
+      zMax: 0.5
+      removeInside: 1
+
+  - BoundingBoxDataPointsFilter:
+      xMin: -6
+      xMax: -1.5
+      yMin: -2.5
+      yMax: 2.5
+      zMin: -1
+      zMax: 1
+      removeInside: 1
+
+  - AddDescriptorDataPointsFilter:
+      descriptorName: omega
+      descriptorDimension: 1
+      descriptorValues: [1]
+
+  - AddDescriptorDataPointsFilter:
+      descriptorName: deviation
+      descriptorDimension: 9
+      descriptorValues: [0.0009, 0, 0, 0, 0.0009, 0, 0, 0, 0.0009]
+
```

Comment:
What are these two descriptors used for?

Reply:
Outdated config, fixed